src/App.vue

```diff
@@ -1,12 +1,24 @@
 <template>
   <div id="app">
-    <h1>BridgeIT</h1>
+    <Navi/>
   </div>
 </template>
 <script>
-export default {};
+import Navi from '@/components/nav.vue';
+export default {
+  name: "App",
+  components: {
+    Navi
+  }
+}
 </script>
 
 <style lang="scss">
 @import "@/assets/style/modules/button.scss";
+body {
```

Comment:
I think these styles were using in globals.scss. Isn't that redundant?

Reply:
i forget delete this.